### DISCUSSION TITLE
FRC Pathing for 2023

### DISCUSSION THREAD

**Discussion by falOn-Dev** [mjansen4857/pathplanner#410] [Q&A]
I recently took over the job of doing autos for my team and our old pathplanner code wasn't done properly so I've needed to rewrite it, but im not sure what the proper way to do autos is because the current wiki is for a beta version as opposed to the stable version. Any code examples would be very helpful

**Comment by mjansen4857** [comment]
Here are the Java usage examples from last year https://github.com/mjansen4857/pathplanner/wiki/PathPlannerLib:-Java-Usage

Be aware that all of this is quite different for 2024 so if you’re trying to get experience for 2024 you might want to check out the beta.

**Reply by falOn-Dev** [reply]
I've been having an issue where when I want to run commands other than a path command the robot code crashes
